library/ssl_tls13_generic.c

```diff
@@ -4783,11 +4846,12 @@ int mbedtls_ssl_parse_new_session_ticket( mbedtls_ssl_context *ssl )
 
     if( suite_info == NULL )
     {
-        MBEDTLS_SSL_DEBUG_MSG( 1, ( "suite_info == NULL, mbedtls_ssl_parse_new_session_ticket failed" ) );
+        MBEDTLS_SSL_DEBUG_MSG( 1, ( "suite_info == NULL, mbedtls_ssl_new_session_ticket_parse failed" ) );
```

Comment:
For assertion failures we so far use blanket the following template
```suggestion
                    MBEDTLS_SSL_DEBUG_MSG( 1, ( "should never happen" ) );
```